cypress/integration/create-task.spec.js

```diff
@@ -0,0 +1,35 @@
+describe("Create a task", () => {
+  beforeEach(() => {
+    cy.setCookie("Other", "other");
+    cy.setCookie("XSRF-TOKEN", "abcde");
+    cy.setCookie("OPG-Bypass-Membrane", "1");
+    cy.visit("/create-task?id=800");
+  });
+
+  it("creates a task for a user", () => {
+    cy.contains("Create Task");
+    cy.get(".moj-banner").should("not.exist");
+    cy.get("#f-taskType").select("Check Application");
+    cy.get("#f-name").type("A title");
+    cy.get("#f-description").type("More words");
+    cy.get("#f-assignTo").check();
```

Comment:
I think this should work, and would make the test clearer to follow:
```suggestion
    cy.contains("User").check();
```